src/transport/smtp/mod.rs

```diff
@@ -178,6 +178,11 @@
 
 use std::time::Duration;
 
+#[cfg(feature = "tokio02")]
+pub use self::async_transport::{
+    AsyncSmtpClient, AsyncSmtpConnector, AsyncSmtpTransport, AsyncSmtpTransportBuilder,
```

Comment:
I'm not sure if we want `Something` and `AsyncSomething` or `Something` and `blocking::Something` (like reqwest does https://docs.rs/reqwest/0.10.7/reqwest/blocking/index.html). Do you know other libs implementing both?

Reply:
I don't have others in mind at the moment. The way reqwest does it is to simply have the blocking versions call the async versions. This is one of the reasons why reqwest is known for being heavy and cause issues in some cases. They import all of the async stuff even when you want to use the blocking versions, because you are always using the async version